### ISSUE TITLE
Should "Swap template" be "Replace template" for consistency?

### ISSUE THREAD

**Issue by bgardner** [WordPress/gutenberg#56845]
### Description

It seems like the word "replace" is used often with action items (replacing template parts, for example). When reviewing https://github.com/WordPress/gutenberg/pull/56407 (which even uses the word "replace" in the issue title), I found it peculiar that the word "swap" is used in the UI. Should we consider "Replace template" > "Swap template"?

### Step-by-step reproduction instructions

1. Go to Site editor > Pages and click a page to edit.
2. On right hand side of screen, click the Templates drop down to view options.

### Screenshots, screen recording, code snippet
![image](https://github.com/WordPress/gutenberg/assets/486261/f73a3ada-44fc-4f8e-887d-52ac4da591a7)

### Environment info

WordPress 6.4.2, Gutenberg 17.2

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by bgardner** [comment]
Or, given some of the work @richtabor has done with removing extra words, should we acknowledge the redundancy in "template" being used, since folks are clicking the Template option and have the drop down menu read:

- Edit
- Replace
- Preview

Rather than:

- Edit template
- Replace template
- Preview template

**Comment by richtabor** [comment]
Makes sense to me. @annezazu I recall you've explored a bit on this popover as well, right?

**Comment by annezazu** [comment]
Yes! Issue here: https://github.com/WordPress/gutenberg/issues/56269 Would be great to update both at once.

**Comment by kohheepeace** [comment]
I prefer **"Change template"** 👍
